src/test/java/com/wikia/webdriver/testsuites/testSuite.xml

```diff
@@ -30,7 +30,6 @@
             <class name="com.wikia.webdriver.testcases.adstests.TestAdsMonitoringOasis"/>
             <class name="com.wikia.webdriver.testcases.adstests.TestAdsMonocolor"/>
             <class name="com.wikia.webdriver.testcases.adstests.TestAdsNoAdsForSony"/>
-            <class name="com.wikia.webdriver.testcases.adstests.TestAdsOnHubs"/>
```

Comment:
no, I checked the jobs defined on Jenkins and grep through jenkins-jobs and jenkins-job-builder repos and there is no mention of hub tests.